temporalio/worker/_workflow_instance.py

```diff
@@ -354,8 +354,8 @@ def activate(
                 elif job.HasField("signal_workflow") or job.HasField("do_update"):
                     job_sets[1].append(job)
                 elif not job.HasField("query_workflow"):
-                    if job.HasField("start_workflow"):
-                        start_job = job.start_workflow
+                    if job.HasField("initialize_workflow"):
```

Comment:
We changed the order this item appears in the activation job list, correct? Can you confirm that that will have no effect here because of the built in Python ordering already present here?

Reply:
Yeah, it doesn't matter because everything gets reordered anyway